src/software/util/time/timestamp.cpp

```diff
@@ -70,3 +71,10 @@ Duration Timestamp::operator-(const Timestamp &timestamp) const
 {
     return Duration::fromSeconds(getSeconds() - timestamp.getSeconds());
 }
+
+std::ostream &operator<<(std::ostream &output_stream, const Timestamp &time)
+{
+    output_stream << std::setprecision(2) << std::fixed << time.getSeconds() << "s";
```

Comment:
Should `2` be a parameter/declared constant? @MathewMacDougall

Reply:
I don't really think so given the context and small usage scope.